awx/ui_next/src/components/CodeMirrorInput/VariablesInput.jsx

```diff
@@ -2,9 +2,9 @@ import React, { useState } from 'react';
 import { string, func, bool, number } from 'prop-types';
 import { Button, Split, SplitItem } from '@patternfly/react-core';
 import styled from 'styled-components';
-import ButtonGroup from '@components/ButtonGroup';
 import { yamlToJson, jsonToYaml, isJson } from '@util/yaml';
 import CodeMirrorInput from './CodeMirrorInput';
+import { ButtonGroup } from '.';
```

Comment:
shouldn't this be ./ButtonGroup?

Reply:
I wondered that too.

Reply:
👍 Since ButtonGroup is only used in /CodeMirrorInput, I'll remove the export from index.js and update the import path in VariablesInput.jsx and YamlJsonToggle.jsx.